crates/search/src/buffer_search.rs

```diff
@@ -399,15 +405,7 @@ impl Render for BufferSearchBar {
                 this.on_action(cx.listener(Self::toggle_whole_word))
             })
             .gap_2()
-            .child(
-                h_flex().child(search_line.w_full()).child(
-                    IconButton::new(SharedString::from("Close"), IconName::Close)
-                        .tooltip(move |cx| Tooltip::for_action("Close search bar", &Dismiss, cx))
-                        .on_click(
-                            cx.listener(|this, _: &ClickEvent, cx| this.dismiss(&Dismiss, cx)),
-                        ),
-                ),
-            )
+            .child(h_flex().child(search_line.w_full()))
```

Comment:
It looks like we can now shorten this line to just `.child(search_line)` since we don't need stretching the line to the full width to pin the close button to the right side.

Reply:
oh yes! I'll adjust and commit now